client/src/components/navbar/index.js

```diff
@@ -0,0 +1,69 @@
+import React ,{Component}from 'react';
+import ReactDOM from 'react-dom';
+import image from  './username.png';
+import plan from  './plan.png';
+import img from  './newTable.png';
+import photo from  './tables.png';
+
+import './style.css'
+
+class Navbar extends Component{
+    constructor() {
+        super();
+        this.state={
+            activeTab: 'flights'
+        }
+
+        this.handleTabChange = this.handleTabChange.bind(this);
+    }
+
+    handleTabChange(activeTab) {
+        this.setState({
+            ...this.state,
+            activeTab
+        })
+    }
+
+    render(){
+        return(
+      <div>
+        <div className = "nav">
+            <div className ="user">
+            <a className="username" href={'#'} >username</a>
+                <img src={image} />
+                
+            </div>
+            <div>
+            <img className ="plane" src={plan} />
+            </div>
+        </div>
+        <div className="sideDiv">
+        <ul>
+            <li
+                onClick={() => this.handleTabChange('flights')}
+                className={this.state.activeTab === 'flights' ? 'active' : ''}
+            >
+            <br></br>
```

Comment:
You could use some css padding or margin to get this spacing instead of the `<br>` tags - That might be more consistent (I think the br's pixel height would change if the line-height or font size of its parent changed).

Reply:
Yes it is true but I put it before I put   padding or margin, I remove it, thanks

Reply:
👍